addons/cookiecutter.json

```diff
@@ -14,6 +14,11 @@
       "path": "./mkdocs",
       "title": "MKDocs",
       "description": "Cookiecutter template for MKDocs"
+    },
+    "disco-like-workflows": {
```

Comment:
I'm not convinced to use the "disco" word. In fact the disco workflows are not like this. The workflows you implement are more workflow to deploy a k8s application from source code so find a name like "k8s-app-workflows" ?

Reply:
I will try to make them as much generic as possible.

Reply:
Check last commit